socli/search.py

```diff
@@ -75,16 +75,16 @@ def get_questions_for_query_google(query, count=10):
     captcha_check(search_results.url)
     soup = BeautifulSoup(search_results.text, 'html.parser')
     try:
-        soup.find_all("div", class_="g")[0]  # For explicitly raising exception
+        soup.find_all("div", class_="ZINbbc xpd O9g5cc uUPGi")[0]  # For explicitly raising exception
     except IndexError:
         socli.printer.print_warning("No results found...")
         sys.exit(0)
-    for result in soup.find_all("div", class_="g"):
+    for result in soup.find_all("div", class_="ZINbbc xpd O9g5cc uUPGi"):
```

Comment:
Class `g` haven't changed. It's still there, You can still use it to select individual results.

Reply:
To get the question title and description, we should rely on the order and hierarchy of the tags under the tag `div` with class `g`.

Reply:
![socli](https://user-images.githubusercontent.com/54910667/94970702-dbd30a00-0522-11eb-8fd2-6a391599f330.jpg)

@gautamkrishnar, Please do give a quick review as when I checked as suggested, using 'g' class returns in no output. At the same time, using the "ZINbbc xpd O9g5cc uUPGi" provided the desired output.

Reply:
@anshik1998 the 'g` class is still available, its working fine for me, please try using this URL: https://www.google.com/search?q=site:stackoverflow.com+python+object+to+json

If you visit the URL via a browser you can see lots of divs like the ones below:
<img width="1440" alt="Screenshot 2020-10-04 at 1 44 37 AM" src="https://user-images.githubusercontent.com/8397274/95000971-3338af00-05e3-11eb-8aeb-49e0070cd270.png">

g is still the enclosing class for all the search lists. At least in the google chrome. In SOCLI we only use google chrome user agents.

Hence the code is working fine for me:
<img width="1440" alt="Screenshot 2020-10-04 at 1 41 19 AM" src="https://user-images.githubusercontent.com/8397274/95000987-62e7b700-05e3-11eb-9eec-a78f545df157.png">